proto/commonfate/access/v1alpha1/context.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package commonfate.access.v1alpha1;
+
+import "commonfate/authz/v1alpha1/read_only.proto";
+
+// The Context API provides contextual information which can be added to access requests.
+service ContextService {
```

Comment:
nit: can we call this `AttachmentService`? `ContextService` might be confusing because we have a `context` already in Cedar authorization requests.